packages/web/src/components/ui/Chip/Chip.module.scss

```diff
@@ -0,0 +1,127 @@
+@import '~styles/_vars.scss';
+@import '~styles/responsive.scss';
+
+.chip {
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 31px;
+  line-height: 1.25;
+  border: 1px solid transparent;
+  white-space: nowrap;
+  transition: all $transition-duration $transition-timing-function;
+
+  @include mobile-and-tablet {
+    font-size: 14px;
+    line-height: 22px;
+  }
+}
+
+.icon {
+  display: inline-flex;
+  margin-right: 2px;
+}
+
+.primary {
+  background-color: $gray-surface;
+
+  &:hover {
+    background-color: $gray-border;
+  }
+}
+
+.secondary {
+  color: $gray-text;
+  background-color: $gray-surface;
+
+  &:hover {
+    background-color: $gray-border;
+  }
+}
+
+.outlined {
+  border: 1px solid $gray-border;
+  background-color: transparent;
+}
+
+.suggest {
+  background: rgba($white, 0.1);
+  color: rgba($white, 0.8);
+}
+
+.badge {
+  padding: 0 9px 0 3px;
+
+  @include mobile-and-tablet {
+    font-size: 15px;
+    line-height: 17px;
+    padding: 2px 10px 2px 2px;
+
+    .icon svg {
+      width: 18px;
+      height: 18px;
+    }
+  }
+}
+
+.default {
+  padding: 4px 11px;
+}
+
+.medium {
+  padding: 7px 11px;
+
+  @include mobile-and-tablet {
+    padding: 4px 11px;
+  }
+}
+
+.large {
+  padding: 15px 19px;
+
+  @include mobile-and-tablet {
+    padding: 12px 15px;
+  }
+}
+
+.sans-serif {
+  font-family: $font-sans-serif;
+}
+
+.monospace {
+  font-family: $font-monospace;
+}
+
+.400 {
```

Comment:
I'm not sure if the selector name is valid since my linter shows an issue on this line